src/redux/actions/authUser.js

```diff
@@ -0,0 +1,33 @@
+// Signup user
+import axios from "axios";
+
+import { SIGNUP_SUCCESS, SIGNUP_ERROR } from "../action_types";
+import { serverError, startFetch } from "./common";
+
+export const signupSuccess = message => ({
+  type: SIGNUP_SUCCESS,
+  message
+});
+
+export const signupError = error => ({
+  type: SIGNUP_ERROR,
+  error
+});
+
+export default user => dispatch => {
```

Comment:
So this thunk is a little bit confusing. Mainly due to the fact that you are also importing it by a different name in the component. Refrain from renaming your actions during import, it becomes harder and harder to track this imports as the project grows.

Refactor the above thunk to:

`export const user = () => 
(dispatch) => {
dispatch(startFetch());
...
}`